transcom/mymove
# B-17689 Fixed use current zip on origin address

## B-17689

## Summary

If a TOO selects “use current ZIP” for the PPM it clears the ZIP. 

## Verification Steps for the Author


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Login to officemove as a TOO
2. Check on the status of a PPM move
3. Click edit shipment
4. Check the 'Use current ZIP' box.
5. Box should populate with origin location zip code.

### Frontend
